packages/dx-react-chart/src/chart.tsx

```diff
@@ -30,13 +43,14 @@ class RawChart extends React.PureComponent<ChartProps> {
       width,
       height,
       children,
+      rotated,
       rootComponent,
       ...restProps
     } = this.props;
     return ((
       <PluginHost>
-        <BasicData data={data} />
-        <Palette scheme={[]} />
+        <BasicData data={data} rotated={rotated!} />
```

Comment:
Should we use `!` here? `rotated` default value is `false`.

Reply:
Yes we should, because `rotated` is declared as optional ans TS doesn't recognize defaultProps yet